### PR TITLE
chore(vale): update word list

### PR DESCRIPTION
**Pull request by dvdksn**
verbiage updates

### CODE REVIEW

File: .github/vale/Vocab/Industry/accept.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 AWS
 Amazon
 Apple
+Azure( Blob Storage)?
```
Comment (usha-mandya) [inline .github/vale/Vocab/Industry/accept.txt:4]: Is the ` ( Blob storage)?` intentional?

Comment (dvdksn) [reply .github/vale/Vocab/Industry/accept.txt:4]: yeah it's an optional (acceptable) suffix to Azure, so it approves both Azure and Azure Blob Storage. We have the same for `GitHub( Actions)?`

Comment (usha-mandya) [reply .github/vale/Vocab/Industry/accept.txt:4]: Makes sense. Thanks

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**